spec/directives/import/error/import_relative.hrx

```diff
@@ -0,0 +1,27 @@
+<===> input.scss
+// Importing a file must not allow that file to resolve its URL
+// relatively to the importer but only to itself (and to load paths)
+@import "child/child"
+
+<===> sibling.scss
+a {b: ""}
+
+<===> child/child.scss
```

Comment:
Naming both path components "child" is a bit confusing. Maybe, `dir/child.scss`?